app/models/user.rb

```diff
@@ -1162,7 +1175,7 @@ def serializable_hash(options = nil)
     # scope at the moment).
     json[:class] = self.class.to_s.downcase
     if include_teams
-      json[:teams] = current_team_members.includes(:team).reject(&:hidden?)
+      json[:teams] = deprecated_team_roles
```

Comment:
Does this change take into account the `hidden` filter that was present in the old implementation?

Reply:
Actually no. I missed it, thanks for pointing. Did it now.